packages/utils/telemetry-utils/src/error.ts

```diff
@@ -53,3 +60,118 @@ export class DataCorruptionError extends LoggingError implements IErrorBase, IFl
 		super(message, { ...props, dataProcessingError: 1 });
 	}
 }
+
+/**
+ * An incoming message from the Fluid Service.
+ */
+export type MessageLike = Partial<
```

Comment:
@alexvy86 @anthony-murphy Any thoughts on the naming and documentation here? I used "MessageLike" as a temporary placeholder.

Reply:
i would prefer it we didn't create an explicity type, and it only existed in the function, as that is the only place it makes sense

Reply:
also, i would mark everything here internal

Reply:
really everything in this package should be internal.

Reply:
Okay. I can re-inline it if it isn't something that stands on its own. I'd still be interested in documenting the type semantics in the two places that use it. Any recommendations for that?

And when you say "mark everything here internal", do you mean these 2 moved items, or **all** of the contents of this module?

Reply:
> really everything in this package should be internal.

Should this package move to the _new_ "@fluid-internal" scope then?

Reply:
that seems wrong, we still need to use it in our public packages, we just don't want to support external usage of the surface area.

Reply:
Tyler is redefining that scope (or creating a new one, I forget where that landed). What I'm referring to is a scope that is published publicly, but documented as not being suitable for external consumption. And that seems to be the intent for this package based on Tyler's spreadsheet.